### ISSUE TITLE
Add config flag to allow multi-account verification for one discord user

### ISSUE THREAD

**Issue by Supposedly-Sam** [DiscordSRV/DiscordSRV#1308]
### Why?

I am an admin for a popular Minecraft server. Our server has discord verification but we wanted to allow people to connect to the server with their alts.

### What and How?

We wonder if it would be possible to implement a feature to allow multi-account verification for one discord user. Optionally, we would like to be able to mark or label the alternative accounts as alts in some way.

### Are there alternatives?

Allow users to label which linked accounts are alts vs main.

### Checks

- [X] I have used the search at least once to check if my idea has already been suggested and perhaps already implemented.

**Comment by DragaoNimhe** [comment]
I would also love this feature, it seems like it would be very useful!

**Comment by RocketManKian** [comment]
Yes! I would love this, would be extremely useful for the DiscordSRV Linking System which forces players to have their MC Account linked to a Discord Account.

**Comment by Vankka** [comment]
> Yes! I would love this, would be extremely useful for the DiscordSRV Linking System which forces players to have their MC Account linked to a Discord Account.

This is already possible, see the options in the linking.yml
